### PR TITLE
Camus architecture diagram and design

### PR DESCRIPTION
**Pull request by Ishiihara**
Added a diagram showing Camus architecture and expand design section in intro.rst

@ewencp @nehanarkhede Can you take a look at this? Thanks!

### CODE REVIEW

**Comment by ewencp** [comment]
Forgot to also mention: looks like something is off with the history. Can you rebase or cherry-pick the one commit so it's just a single commit on top of the current confluent-master?

File: docs/intro.rst [hunk 1]
```diff
@@ -37,12 +39,19 @@ Key Features
 
 Design
```
Comment (nehanarkhede) [inline docs/intro.rst]: I think @ewencp mentioned this too. I think we can put this after the intro and config and before the operations

File: docs/intro.rst [hunk 2]
```diff
@@ -83,11 +78,31 @@ Hadoop in pseudo-distributed mode, see this
...
+   See this
+   `guide <http://www.cloudera.com/content/cloudera/en/documentation/core/latest/topics/cdh_qs_cdh5_pseudo.html>`_.
+   to setup a single Hadoop node in pseudo-distributed mode.
+
+.. ifconfig:: not platform_docs
+
+   You can download prebuilt versions of Camus as part of the
+   `Confluent Platform <http://confluent.io/downloads/>`_. To install from
+   source, follow the instructions in the `Development`_ section.
+   Before starting a Camus job you must have Haddop, Kafka, and the
```
Comment (nehanarkhede) [inline docs/intro.rst]: This looks great. Let's link to the place that tells users how to start Kafka and Schema Registry. Currently, it says "the Schema Registry repository" which seems a little awkward. @ewencp Does this sort of cross repo linking have to wait until the docs are published?

**Comment by nehanarkhede** [comment]
@Ishiihara Thanks for adding the picture! Few comments-
1. The resolution should be fixed. 
2. Let's remove the shadow for each of the boxes. I'm assuming you used Omnigraffle. 
3. I agree with @ewencp that we can remove the ZooKeeper box as well as the Metadata
4. We need to think what the boxes signify. For example, the box around all the ETL Map Task looks like the Camus job but there is a separate Camus box outside the Hadoop Cluster one. I think you meant to show the job startup vs job execution? If so, let's name the boxes differently. 
5. Let's standardize on the font. I don't have a preference, I've used Helvetica Neue for the box labels for the main deployment picture. 
6. The box sizes could be fixed a little. For example, the Schema Registry box is too large compared to the Kafka, ZooKeeper and Camus ones.

In general, I think working on feedback for the image is tricky without a face-to-face chat. Let's discuss tomorrow morning.

**Comment by nehanarkhede** [comment]
Minor nit - The subsection under "Camus Operations" could be renamed from "Operations Related Configurations" to "Important Configuration Options". Since it is already under the Operations section, it is implied that those notes are operations related

**Comment by Ishiihara** [comment]
@ewencp @nehanarkhede Addressed review comments on the diagram and texts. Would you please do a second round review?

File: docs/intro.rst [hunk 3]
```diff
@@ -6,7 +6,8 @@ It is capable of incrementally copying data form Kafka into HDFS such that
 every run of the MapReduce job picks up where the previous run left off.
 At LinkedIn, Camus is used to load billions of messages per day from Kafka into HDFS.
 Confluent's version of Camus integrates with Confluent's Schema Registry which
-ensures data compatibility when loading to HDFS as schemas are evolved.
+ensures data compatibility when loading to HDFS as schemas are evolved. You can find the design
+and architecture of Camus in :ref:`design section<camus_design>` .
```
Comment (ewencp) [inline docs/intro.rst]: in design section -> in the design section

**Comment by ewencp** [comment]
@Ishiihara The image is much improved. I had one minor nit, but otherwise LGTM.

**Comment by Ishiihara** [comment]
@ewencp @nehanarkhede Let me if you have further comments on this PR? Thanks!

**Comment by ewencp** [comment]
@Ishiihara Looks good to me. I think the only thing missing is the source (omnigraffle or whatever you used) file for the image.

**Pull request merged**